src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -5697,7 +5698,7 @@ public void IOFailure_MovePdbTemp()
             MockCSharpCompiler csc = MakeTrackingCsc(tempFilePaths, "/nologo", "/debug", "/preferreduilang:en", sourcePath);
             csc.FileMove = (source, dest) =>
             {
-                if (dest.EndsWith(".pdb"))
+                if (dest.EndsWith(".pdb", StringComparison.OrdinalIgnoreCase))
```

Comment:
I don't think we should. It has to be case by case. It won't work in general - I think I already gave an example of a case-insensitive FS mounted to a directory on Linux.